### ISSUE TITLE
run VEST_PA_18_replication.ipynb

### ISSUE THREAD

**Issue by rootsmusic** [nonpartisan-redistricting-datahub/pdv-pa#8]
@peter-redistricting-data-hub  (I don't have PA's raw-from-source files anymore,)  It'd be nice to update the Validation Report.  Thanks.

P.S. I assume that PA_18_ERJ.ipynb sourced from [VEST's updated file](https://dataverse.harvard.edu/file.xhtml?fileId=6880744&version=64.0), which has updated its precinct boundaries and swapped a couple precincts.

**Comment by peter-redistricting-data-hub** [comment]
Yup, we updated the ERJ file using VEST's updated file. 

And you are taking about the PDV report?

**Comment by rootsmusic** [comment]
Yes @peter-redistricting-data-hub, the [Validation Report](https://redistrictingdatahub.org/wp-content/uploads/2020/12/pa_vest_18_validation_report.pdf) because VEST's metadata didn't highlight what's been updated in their file.

P.S. when you were working on PA_18_ERJ.ipynb, [my issue](https://github.com/nonpartisan-redistricting-datahub/pdv-pa/issues/5) alerted you about Delaware's errata.  If your notebook had already corrected my issue, then it shouldn't swap the corrected precincts again when VEST realizes that issue.
